components/HelloWave.tsx

```
@@ -26,7 +25,7 @@ export function HelloWave() {
 
   return (
     <Animated.View style={animatedStyle}>
-      <ThemedText style={styles.text}>👋</ThemedText>
+      <Text style={styles.text}>👋</Text>
```

ESLint

[error] 'Text' is not defined.

(react/jsx-no-undef)